app/controllers/front_ui_controller.rb

```diff
@@ -44,6 +44,20 @@ def save_location
 
   def about; end
 
+  def location
+    @loc = params[:loc].downcase
```

Comment:
```suggestion
    @loc = params[:loc]
```

This will not be required as it will not consider the case